### PR TITLE
feat: implemented chain specific overview

### PR DESCRIPTION
**Pull request by Teja2045**
#879

### CODE REVIEW

**Comment by aleem1314** [comment]
Missing navigation from overview page to chain specific page

**Comment by aleem1314** [comment]
chain-specific
balance

**Comment by Teja2045** [comment]
> https://github-production-user-asset-6210df.s3.amazonaws.com/72057206/284846038-6bd11436-7977-4a39-abe7-74a9a85c0eb8.png

done

File: frontend/src/utils/copyToClipboard.ts [hunk 1]
```diff
@@ -0,0 +1,3 @@
+export const copyToClipboard = (text: string) => {
+  navigator.clipboard.writeText(text);
+};
```
Comment (aleem1314) [inline frontend/src/utils/copyToClipboard.ts:3]:
```suggestion
export const copyToClipboard = (text: string) => navigator.clipboard.writeText(text);
```

File: frontend/src/app/(routes)/(overview)/overview-components/WalletSummery.tsx [hunk 2]
```diff
@@ -1,33 +1,32 @@
...
       alt: 'rewards',
       type: 'Rewards',
       amount: rewards,
     },
     {
-      icon: 'balanceAmount.svg',
+      icon: '/balanceAmount.svg',
       alt: 'available',
-      type: 'Wallet Balance',
+      type: 'Available Balance',
```
Comment (aleem1314) [inline frontend/src/app/(routes)/(overview)/overview-components/WalletSummery.tsx:29]:
```suggestion
      type: 'Available',
```

File: frontend/src/app/(routes)/(overview)/overview/[...chainNames]/page.tsx [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+      {chainIDs.length ? (
+        <OverviewPage chainIDs={chainIDs} />
+      ) : (
+        <div className="w-full h-full text-white flex justify-center items-center">
+          - Chain Not found -
+        </div>
+      )}
+    </>
+  );
+  //return <>Overview</>
```
Comment (aleem1314) [inline frontend/src/app/(routes)/(overview)/overview/[...chainNames]/page.tsx:35]:
```suggestion
```

File: frontend/src/app/(routes)/(overview)/page.tsx [hunk 4]
```diff
@@ -1,11 +1,20 @@
...
-   return <OverviewPage />;
+const Overview = () => {
+  const nameToChainIDs = useAppSelector(
+    (state: RootState) => state.wallet.nameToChainIDs
+  );
+  const chainIDs = Object.keys(nameToChainIDs).map(
+    (chainName) => nameToChainIDs[chainName]
+  );
+  return <OverviewPage chainIDs={chainIDs} />;
   //return <>Overview</>
```
Comment (aleem1314) [inline frontend/src/app/(routes)/(overview)/page.tsx:17]:
```suggestion
```

File: frontend/src/components/CopyToClipboard.tsx [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+
+export const CopyToClipboard = ({
+  message,
+  formattedMessage,
+}: {
+  message: string;
+  formattedMessage: string;
+}) => {
+  return (
+    <Box>
```
Comment (aleem1314) [inline frontend/src/components/CopyToClipboard.tsx:14]:
```suggestion
```

File: frontend/src/components/CopyToClipboard.tsx [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+        label={formattedMessage}
+        size="small"
+        deleteIcon={
+          <Image src="/copy.svg" height={24} width={24} alt="copy-content" />
+        }
+        onDelete={() => {
+          copyToClipboard(message);
+        }}
+      />
+    </Box>
```
Comment (aleem1314) [inline frontend/src/components/CopyToClipboard.tsx:26]:
```suggestion
```

**Review by aleem1314 — APPROVED** [review]

**Pull request merged**